### PR TITLE
Allow the website to display seed ID on file select

### PR DESCRIPTION
**Pull request by fenhl**
This adds a hidden `web_id` setting which can be set to a number by a future version of ootrandomizer.com, in which case it will be displayed on the file select screen:

image

The song title display is moved out of the way to the bottom of the file select screen:

image

### CODE REVIEW

File: ASM/src/config.asm [hunk 1]
```diff
@@ -123,6 +123,11 @@ WORLD_STRING_TXT:
 TIME_STRING_TXT:
 .endarea
 
+; web seed ID string (max length 10 chars)
+.area 0x10, 0
```
Comment (cjohnson57) [inline ASM/src/config.asm:127]: Since it's 10 chars should this be 0x0A?

Comment (fenhl) [reply ASM/src/config.asm:127]: `0x0B` technically, to account for the null terminator. I just made it match the `WORLD_STRING_TXT` symbol above which is also 10 chars.

Comment (cjohnson57) [reply ASM/src/config.asm:127]: Fair enough then

**Review by cjohnson57 — APPROVED** [review]

**Comment by fenhl** [comment]
@TreZc0 @dragonbane0 do you have any way to test this with the website? Or do you think it makes more sense to do that after merging?

**Comment by TreZc0** [comment]
Yea i will test it asap

**Comment by TreZc0** [comment]
Can confirm this works just fine.

**Pull request merged**